tests/test_models/test_engine/test_file_storage.py

```diff
@@ -113,3 +113,34 @@ def test_save(self):
         with open("file.json", "r") as f:
             js = f.read()
         self.assertEqual(json.loads(string), json.loads(js))
+
+    @unittest.skipIf(models.storage_t == 'db', "testing file storage")
+    def test_get(self):
+        """Test that get retrieves a specific object"""
+        state = State(name="Virginia")
+        state.save()
+        response = models.storage.get(State, state.id)
+        self.assertEqual(response.id, state.id)
```

Comment:
Add a test that checks when the string is passed to storage.get() function.
Example:
1. storage.get(State, state.id)
2. storage.all("State", state.id)
In the second point must fail